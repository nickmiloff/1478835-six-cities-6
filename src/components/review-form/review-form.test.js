import React from 'react';
import {render} from '@testing-library/react';
import Test from '../test/test';

import ReviewForm from './review-form';

describe(`ReviewForm component test`, () => {
  it(`ReviewForm component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <ReviewForm />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
