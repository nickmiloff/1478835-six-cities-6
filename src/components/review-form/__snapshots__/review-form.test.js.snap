// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewForm component test ReviewForm component' should render correctly 1`] = `
<div>
  <form
    action="#"
    class="reviews__form form"
    data-testid="form"
    method="post"
  >
    <label
      class="reviews__label form__label"
      for="review"
    >
      Your review
    </label>
    <div
      class="reviews__rating-form form__rating"
    >
      <input
        class="form__rating-input visually-hidden"
        id="5-stars"
        name="rating"
        type="radio"
        value="5"
      />
      <label
        class="reviews__rating-label form__rating-label"
        for="5-stars"
        title="perfect"
      >
        <svg
          class="form__star-image"
          height="33"
          width="37"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
      </label>
      <input
        class="form__rating-input visually-hidden"
        id="4-stars"
        name="rating"
        type="radio"
        value="4"
      />
      <label
        class="reviews__rating-label form__rating-label"
        for="4-stars"
        title="perfect"
      >
        <svg
          class="form__star-image"
          height="33"
          width="37"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
      </label>
      <input
        class="form__rating-input visually-hidden"
        id="3-stars"
        name="rating"
        type="radio"
        value="3"
      />
      <label
        class="reviews__rating-label form__rating-label"
        for="3-stars"
        title="perfect"
      >
        <svg
          class="form__star-image"
          height="33"
          width="37"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
      </label>
      <input
        class="form__rating-input visually-hidden"
        id="2-stars"
        name="rating"
        type="radio"
        value="2"
      />
      <label
        class="reviews__rating-label form__rating-label"
        for="2-stars"
        title="perfect"
      >
        <svg
          class="form__star-image"
          height="33"
          width="37"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
      </label>
      <input
        class="form__rating-input visually-hidden"
        id="1-stars"
        name="rating"
        type="radio"
        value="1"
      />
      <label
        class="reviews__rating-label form__rating-label"
        for="1-stars"
        title="perfect"
      >
        <svg
          class="form__star-image"
          height="33"
          width="37"
        >
          <use
            xlink:href="#icon-star"
          />
        </svg>
      </label>
    </div>
    <textarea
      class="reviews__textarea form__textarea"
      data-testid="review"
      id="review"
      maxlength="300"
      name="review"
      placeholder="Tell how was your stay, what you like and what can be improved"
    />
    <div
      class="reviews__button-wrapper"
    >
      <p
        class="reviews__help"
      >
        To submit review please make sure to set 
        <span
          class="reviews__star"
        >
          rating
        </span>
         and describe your stay with at least 
        <b
          class="reviews__text-amount"
        >
          50 characters
        </b>
        .
      </p>
      <button
        class="reviews__submit form__submit button"
        disabled=""
        type="submit"
      >
        Submit
      </button>
    </div>
  </form>
</div>
`;
